{"context": "Tensorflow\u304cWindows\u306e\u30cd\u30a4\u30c6\u30a3\u30d6\u30b5\u30dd\u30fc\u30c8\u3092\u3057\u305f\u306e\u3067\u30a4\u30f3\u30b9\u30c8\u30fc\u30eb\n\u3053\u3061\u3089\u306e\u7d9a\u304d\u306b\u306a\u308a\u307e\u3059\u3002\n\n\u3053\u3061\u3089\u306e\u74b0\u5883\u306f\u3001\u6700\u65b0(AnniversaryUpdate\u6e08\u307f)Windows10\u300164bit\u3067\u3059\u3002\n\u3055\u3066\u3001\u524d\u56de\u30a4\u30f3\u30b9\u30c8\u30fc\u30eb\u306b\u6210\u529f\u3057\u307e\u3057\u305f\u3002\nPython\u3067\u5b9f\u969b\u306b\u30a4\u30f3\u30dd\u30fc\u30c8\u306b\u6311\u6226\u3057\u3066\u307f\u307e\u3059\u3002\npython\u30b3\u30de\u30f3\u30c9\u3092\u5165\u529b\u3059\u308b\u3068\u3001\u5bfe\u8a71\u578b\u306e\u30d7\u30ed\u30b0\u30e9\u30e0\u304c\u5165\u529b\u3067\u304d\u307e\u3059\u3002\n\u7279\u306b\u8a2d\u5b9a\u3092\u3057\u306a\u3044\u3068\u3001python.exe\u306f\u3053\u3053\u306b\u3042\u308b\u306f\u305a\u3002\nC:\\Users\\\u30e6\u30fc\u30b6\u540d\\AppData\\Local\\Programs\\Python\\Python35\n\n\u3053\u306e\u30c7\u30a3\u30ec\u30af\u30c8\u30ea\u3092\u74b0\u5883\u5909\u6570Path\u306b\u8ffd\u52a0\u3057\u3066\u304a\u3044\u3066\u304f\u3060\u3055\u3044\u3002\n>>> import tensorflow\nTraceback (most recent call last):\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 18, in swig_import_helper\n    return importlib.import_module(mname)\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\importlib\\__init__.py\", line 126, in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\n  File \"<frozen importlib._bootstrap>\", line 986, in _gcd_import\n  File \"<frozen importlib._bootstrap>\", line 969, in _find_and_load\n  File \"<frozen importlib._bootstrap>\", line 958, in _find_and_load_unlocked\n  File \"<frozen importlib._bootstrap>\", line 666, in _load_unlocked\n  File \"<frozen importlib._bootstrap>\", line 577, in module_from_spec\n  File \"<frozen importlib._bootstrap_external>\", line 906, in create_module\n  File \"<frozen importlib._bootstrap>\", line 222, in _call_with_frames_removed\nImportError: DLL load failed: \u6307\u5b9a\u3055\u308c\u305f\u30e2\u30b8\u30e5\u30fc\u30eb\u304c\u898b\u3064\u304b\u308a\u307e\u305b\u3093\u3002\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\__init__.py\", line 54, in <module>\n    from tensorflow.python import pywrap_tensorflow\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 21, in <module>\n    _pywrap_tensorflow = swig_import_helper()\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 20, in swig_import_helper\n    return importlib.import_module('_pywrap_tensorflow')\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\importlib\\__init__.py\", line 126, in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\nImportError: No module named '_pywrap_tensorflow'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"<stdin>\", line 1, in <module>\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\__init__.py\", line 24, in <module>\n    from tensorflow.python import *\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\__init__.py\", line 60, in <module>\n    raise ImportError(msg)\nImportError: Traceback (most recent call last):\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 18, in swig_import_helper\n    return importlib.import_module(mname)\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\importlib\\__init__.py\", line 126, in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\n  File \"<frozen importlib._bootstrap>\", line 986, in _gcd_import\n  File \"<frozen importlib._bootstrap>\", line 969, in _find_and_load\n  File \"<frozen importlib._bootstrap>\", line 958, in _find_and_load_unlocked\n  File \"<frozen importlib._bootstrap>\", line 666, in _load_unlocked\n  File \"<frozen importlib._bootstrap>\", line 577, in module_from_spec\n  File \"<frozen importlib._bootstrap_external>\", line 906, in create_module\n  File \"<frozen importlib._bootstrap>\", line 222, in _call_with_frames_removed\nImportError: DLL load failed: \u6307\u5b9a\u3055\u308c\u305f\u30e2\u30b8\u30e5\u30fc\u30eb\u304c\u898b\u3064\u304b\u308a\u307e\u305b\u3093\u3002\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\__init__.py\", line 54, in <module>\n    from tensorflow.python import pywrap_tensorflow\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 21, in <module>\n    _pywrap_tensorflow = swig_import_helper()\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 20, in swig_import_helper\n    return importlib.import_module('_pywrap_tensorflow')\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\importlib\\__init__.py\", line 126, in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\nImportError: No module named '_pywrap_tensorflow'\n\n\nError importing tensorflow.  Unless you are using bazel,\nyou should not try to import tensorflow from its source directory;\nplease exit the tensorflow source tree, and relaunch your python interpreter\nfrom there.\n>>>\n\n\u9577\u3059\u304e\u308b\uff01\nPython\u306b\u3082\u8a73\u3057\u304f\u7121\u3044\u306e\u3067\u30a8\u30e9\u30fc\u306e\u898b\u65b9\u3082\u308f\u304b\u3089\u306a\u3044\u3002\n\u3068\u3044\u3046\u3053\u3068\u3067\u3001\u691c\u7d22\u3057\u3066\u307f\u307e\u3057\u305f\u3002\n\u30a8\u30e9\u30fc\u30e1\u30c3\u30bb\u30fc\u30b8\u304b\u3089\u4e00\u884c\u629c\u304d\u51fa\u3057\u3066\u3001\n#google\u691c\u7d22\u300cError importing tensorflow.  Unless you are using bazel, \"windows\"\u300d\n\u2193\ngithub\u3067\u305d\u308c\u3063\u307d\u3044\u3082\u306e\u3092\u767a\u898b\u3002\nhttps://github.com/tensorflow/tensorflow/issues/5949\nmrry\u3055\u3093\u306e\u56de\u7b54\u3092\u53c2\u8003\u306b\u3055\u305b\u3066\u3044\u305f\u3060\u304d\u307e\u3057\u305f\u3002\nMicrosoft Visual C++ 2015 Redistributable Update 3\n\u3053\u3061\u3089\u306eexe\u30d5\u30a1\u30a4\u30eb\u3092\u30c0\u30a6\u30f3\u30ed\u30fc\u30c9\u3057\u3066\u5b9f\u884c\u3057\u3066\u3001\u3082\u3046\u4e00\u5ea6import\u3092\u5b9f\u884c\u3057\u3066\u307f\u307e\u3059\u3002\n>>> import tensorflow\n>>>\n\n\u4e0a\u624b\u304f\u3044\u304d\u307e\u3057\u305f\u3002\n\u3053\u3053\u307e\u3067\u6765\u305f\u3089\u5f8c\u306f\u7c21\u5358\u3002\n\u5358\u7d14\u306a\u30d7\u30ed\u30b0\u30e9\u30e0\u3092\u52d5\u304b\u3057\u3066\u307f\u307e\u3059\u3002\n>>> import tensorflow as tf\n>>> hello = tf.constant('Hello tensorflow')\n>>> sess = tf.Session()\n>>> print(sess.run(hello))\nb'Hello tensorflow'\n>>>\n\n\u52d5\u304d\u307e\u3057\u305f\uff01\n\u4eca\u56de\u306f\u30a8\u30e9\u30fc\u30e1\u30c3\u30bb\u30fc\u30b8\u3067\u76f4\u63a5\u691c\u7d22\u3057\u305f\u3089\u89e3\u6c7a\u3057\u305f\u306e\u3067\u3001\u4ed6\u306e\u7a2e\u985e\u306e\u30a8\u30e9\u30fc\u304c\u51fa\u305f\u65b9\u3067\u3082\u76f4\u63a5\u691c\u7d22\u3067\u306a\u3093\u3068\u304b\u306a\u308b\u304b\u3082\u3057\u308c\u307e\u305b\u3093\u306d\u3002\n\u3053\u308c\u3067\u3064\u3044\u306b\u6a5f\u68b0\u5b66\u7fd2\u306b\u6311\u6226\u3067\u304d\u305d\u3046\u3067\u3059\u3002\n[Tensorflow\u304cWindows\u306e\u30cd\u30a4\u30c6\u30a3\u30d6\u30b5\u30dd\u30fc\u30c8\u3092\u3057\u305f\u306e\u3067\u30a4\u30f3\u30b9\u30c8\u30fc\u30eb](http://qiita.com/intelf___/items/7a6eb39cd4efa94e4987)\n\u3053\u3061\u3089\u306e\u7d9a\u304d\u306b\u306a\u308a\u307e\u3059\u3002\n\n---\n\u3053\u3061\u3089\u306e\u74b0\u5883\u306f\u3001\u6700\u65b0(AnniversaryUpdate\u6e08\u307f)Windows10\u300164bit\u3067\u3059\u3002\n\n\u3055\u3066\u3001\u524d\u56de\u30a4\u30f3\u30b9\u30c8\u30fc\u30eb\u306b\u6210\u529f\u3057\u307e\u3057\u305f\u3002\nPython\u3067\u5b9f\u969b\u306b\u30a4\u30f3\u30dd\u30fc\u30c8\u306b\u6311\u6226\u3057\u3066\u307f\u307e\u3059\u3002\npython\u30b3\u30de\u30f3\u30c9\u3092\u5165\u529b\u3059\u308b\u3068\u3001\u5bfe\u8a71\u578b\u306e\u30d7\u30ed\u30b0\u30e9\u30e0\u304c\u5165\u529b\u3067\u304d\u307e\u3059\u3002\n\u7279\u306b\u8a2d\u5b9a\u3092\u3057\u306a\u3044\u3068\u3001python.exe\u306f\u3053\u3053\u306b\u3042\u308b\u306f\u305a\u3002\n\n```\nC:\\Users\\\u30e6\u30fc\u30b6\u540d\\AppData\\Local\\Programs\\Python\\Python35\n```\n\n\u3053\u306e\u30c7\u30a3\u30ec\u30af\u30c8\u30ea\u3092\u74b0\u5883\u5909\u6570Path\u306b\u8ffd\u52a0\u3057\u3066\u304a\u3044\u3066\u304f\u3060\u3055\u3044\u3002\n\n```\n>>> import tensorflow\nTraceback (most recent call last):\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 18, in swig_import_helper\n    return importlib.import_module(mname)\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\importlib\\__init__.py\", line 126, in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\n  File \"<frozen importlib._bootstrap>\", line 986, in _gcd_import\n  File \"<frozen importlib._bootstrap>\", line 969, in _find_and_load\n  File \"<frozen importlib._bootstrap>\", line 958, in _find_and_load_unlocked\n  File \"<frozen importlib._bootstrap>\", line 666, in _load_unlocked\n  File \"<frozen importlib._bootstrap>\", line 577, in module_from_spec\n  File \"<frozen importlib._bootstrap_external>\", line 906, in create_module\n  File \"<frozen importlib._bootstrap>\", line 222, in _call_with_frames_removed\nImportError: DLL load failed: \u6307\u5b9a\u3055\u308c\u305f\u30e2\u30b8\u30e5\u30fc\u30eb\u304c\u898b\u3064\u304b\u308a\u307e\u305b\u3093\u3002\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\__init__.py\", line 54, in <module>\n    from tensorflow.python import pywrap_tensorflow\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 21, in <module>\n    _pywrap_tensorflow = swig_import_helper()\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 20, in swig_import_helper\n    return importlib.import_module('_pywrap_tensorflow')\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\importlib\\__init__.py\", line 126, in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\nImportError: No module named '_pywrap_tensorflow'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"<stdin>\", line 1, in <module>\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\__init__.py\", line 24, in <module>\n    from tensorflow.python import *\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\__init__.py\", line 60, in <module>\n    raise ImportError(msg)\nImportError: Traceback (most recent call last):\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 18, in swig_import_helper\n    return importlib.import_module(mname)\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\importlib\\__init__.py\", line 126, in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\n  File \"<frozen importlib._bootstrap>\", line 986, in _gcd_import\n  File \"<frozen importlib._bootstrap>\", line 969, in _find_and_load\n  File \"<frozen importlib._bootstrap>\", line 958, in _find_and_load_unlocked\n  File \"<frozen importlib._bootstrap>\", line 666, in _load_unlocked\n  File \"<frozen importlib._bootstrap>\", line 577, in module_from_spec\n  File \"<frozen importlib._bootstrap_external>\", line 906, in create_module\n  File \"<frozen importlib._bootstrap>\", line 222, in _call_with_frames_removed\nImportError: DLL load failed: \u6307\u5b9a\u3055\u308c\u305f\u30e2\u30b8\u30e5\u30fc\u30eb\u304c\u898b\u3064\u304b\u308a\u307e\u305b\u3093\u3002\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\__init__.py\", line 54, in <module>\n    from tensorflow.python import pywrap_tensorflow\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 21, in <module>\n    _pywrap_tensorflow = swig_import_helper()\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\tensorflow\\python\\pywrap_tensorflow.py\", line 20, in swig_import_helper\n    return importlib.import_module('_pywrap_tensorflow')\n  File \"C:\\Users\\a1503\\AppData\\Local\\Programs\\Python\\Python35\\lib\\importlib\\__init__.py\", line 126, in import_module\n    return _bootstrap._gcd_import(name[level:], package, level)\nImportError: No module named '_pywrap_tensorflow'\n\n\nError importing tensorflow.  Unless you are using bazel,\nyou should not try to import tensorflow from its source directory;\nplease exit the tensorflow source tree, and relaunch your python interpreter\nfrom there.\n>>>\n```\n\n\u9577\u3059\u304e\u308b\uff01\n\nPython\u306b\u3082\u8a73\u3057\u304f\u7121\u3044\u306e\u3067\u30a8\u30e9\u30fc\u306e\u898b\u65b9\u3082\u308f\u304b\u3089\u306a\u3044\u3002\n\u3068\u3044\u3046\u3053\u3068\u3067\u3001\u691c\u7d22\u3057\u3066\u307f\u307e\u3057\u305f\u3002\n\u30a8\u30e9\u30fc\u30e1\u30c3\u30bb\u30fc\u30b8\u304b\u3089\u4e00\u884c\u629c\u304d\u51fa\u3057\u3066\u3001\n**#google\u691c\u7d22\u300cError importing tensorflow.  Unless you are using bazel, \"windows\"\u300d**\n\n\u2193\n\ngithub\u3067\u305d\u308c\u3063\u307d\u3044\u3082\u306e\u3092\u767a\u898b\u3002\nhttps://github.com/tensorflow/tensorflow/issues/5949\n***mrry***\u3055\u3093\u306e\u56de\u7b54\u3092\u53c2\u8003\u306b\u3055\u305b\u3066\u3044\u305f\u3060\u304d\u307e\u3057\u305f\u3002\n\n[Microsoft Visual C++ 2015 Redistributable Update 3](https://www.microsoft.com/en-us/download/details.aspx?id=53587)\n\u3053\u3061\u3089\u306eexe\u30d5\u30a1\u30a4\u30eb\u3092\u30c0\u30a6\u30f3\u30ed\u30fc\u30c9\u3057\u3066\u5b9f\u884c\u3057\u3066\u3001\u3082\u3046\u4e00\u5ea6import\u3092\u5b9f\u884c\u3057\u3066\u307f\u307e\u3059\u3002\n\n```\n>>> import tensorflow\n>>>\n```\n\u4e0a\u624b\u304f\u3044\u304d\u307e\u3057\u305f\u3002\n\n\u3053\u3053\u307e\u3067\u6765\u305f\u3089\u5f8c\u306f\u7c21\u5358\u3002\n\u5358\u7d14\u306a\u30d7\u30ed\u30b0\u30e9\u30e0\u3092\u52d5\u304b\u3057\u3066\u307f\u307e\u3059\u3002\n\n```\n>>> import tensorflow as tf\n>>> hello = tf.constant('Hello tensorflow')\n>>> sess = tf.Session()\n>>> print(sess.run(hello))\nb'Hello tensorflow'\n>>>\n```\n\n\u52d5\u304d\u307e\u3057\u305f\uff01\n\u4eca\u56de\u306f\u30a8\u30e9\u30fc\u30e1\u30c3\u30bb\u30fc\u30b8\u3067\u76f4\u63a5\u691c\u7d22\u3057\u305f\u3089\u89e3\u6c7a\u3057\u305f\u306e\u3067\u3001\u4ed6\u306e\u7a2e\u985e\u306e\u30a8\u30e9\u30fc\u304c\u51fa\u305f\u65b9\u3067\u3082\u76f4\u63a5\u691c\u7d22\u3067\u306a\u3093\u3068\u304b\u306a\u308b\u304b\u3082\u3057\u308c\u307e\u305b\u3093\u306d\u3002\n\u3053\u308c\u3067\u3064\u3044\u306b\u6a5f\u68b0\u5b66\u7fd2\u306b\u6311\u6226\u3067\u304d\u305d\u3046\u3067\u3059\u3002\n", "tags": ["TensorFlow", "Windows", "Windows10", "Python"]}